rfcs/20211025-augru/README.md

```diff
@@ -0,0 +1,1089 @@
+Proposal for GRU with Attentional Update in DNNL
+================================================
+
+## 1. Introduction
+
+In addition to the regular GRU there is a notion of GRU with attentional update
+gate (AUGRU) that can help to model interest evolving in Click-Through Rate
+Prediction [[#1]][1].
+
+
+### 1.1. GRU
+
+GRU is defined by the following formula:
+
+![GRU-LBR](formula-gru-lbr.png)
```

Comment:
(Very minor) for some reason this formula is generated on transparent background, so the formula is not visible in dark mode :-)

Reply:
Added a background, please check